docs/how_to/distributed_training.md

```diff
@@ -0,0 +1,288 @@
+# Distributed training
+MXNet supports distributed training enabling us to leverage multiple machines for faster training.
+In this document, we describe how it works, how to launch a distributed training job and
+some environment variables which provide more control.
+
+## Type of parallelism
+
+There are two ways in which we can distribute the workload of training a neural network across multiple devices (can be either GPU or CPU).
+The first way is *data parallelism*, which refers to the case where each device stores a complete copy of the model.
+Each device works with a different part of the dataset, and the devices collectively update a shared model.
```

Comment:
Can we rephrase this "devices collectively update a shared model" better? One possible I can think of is "the devices update their model and share these updates with other devices who apply them so that the model is in sync across devices."?

Reply:
Will think about how to make this line better tomorrow. I agree rephrasing would be better